pkg/skaffold/event/event.go

```diff
@@ -363,13 +366,15 @@ func (ev *eventHandler) handleFileSyncEvent(e *proto.FileSyncEvent) {
 	})
 }
 
-func LogSkaffoldMetadata(info *version.Info) {
+func LogMetaEvent(info *version.Info) {
+	metadata := handler.state.Metadata
```

Comment:
nit: I would just call `version.Get()` in this function, and then have this take no parameters.